Lottie.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import LottieView from "lottie-react-native";
+
+export const Lottie = () => {
```

Comment:
maybe this component should be called Virus or something like that? To more explain the component